src/clj/puppetlabs/services/jruby/jruby_puppet_internal.clj

```diff
@@ -100,7 +96,6 @@
   ; Note, this behavior should remain consistent with new-main
   (doto scripting-container
     (.setLoadPaths (managed-load-path ruby-load-path))
-    (.setCompatVersion compat-version)
```

Comment:
Ruby version is hardcoded to "2.2" in JRuby 9K.  The `ScriptingContainer` still supports the `.setCompatVersion` method but it's deprecated and does nothing - https://github.com/jruby/jruby/blob/9.0.4.0/core/src/main/java/org/jruby/embed/ScriptingContainer.java#L503-L505.  Seemed best to just not call it anymore.